website/docs/providers/rac/how-to-rac.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Create a Remote Access (RAC) provider
+---
+
+:::info
+This feature is in technical preview, so please report any Bugs you run into on [GitHub](https://github.com/goauthentik/authentik/issues)
```

Comment:
Pretty sure "bugs" only needs to be capitalized when talking about this guy: 
![image](https://github.com/goauthentik/authentik/assets/133134217/90a66922-6427-430b-9543-51bcd95be22b)